### PR TITLE
Explain how to generate a safe Encryption key

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/Security/GuidelinesIntegrators/InstallTool.rst [hunk 1]
```diff
@@ -150,5 +150,22 @@ information, cache content, etc. and you should clear all caches after
...
 the new encryption key.
 
-Keep in mind that this string is security-related and you should keep
-it in a safe place.
+..  attention::
+    Keep in mind that this string is security-related and you should keep
+    it in a safe place.
+
+The encryption key should be a random hexadecimal key of length 96. You can 
+for example create it with openssl:
```
Comment (brotkrueml) [inline Documentation/Security/GuidelinesIntegrators/InstallTool.rst:158]:
```suggestion
for example create it with OpenSSL:
```

File: Documentation/Security/GuidelinesIntegrators/InstallTool.rst [hunk 2]
```diff
@@ -150,5 +150,22 @@ information, cache content, etc. and you should clear all caches after
...
+
+..  code-block:: bash
+
+    openssl rand -hex 48 
+    
+From withing TYPO3 it is possible to generate it via API:
```
Comment (brotkrueml) [inline Documentation/Security/GuidelinesIntegrators/InstallTool.rst:164]:
```suggestion
From within TYPO3 it is possible to generate it via API:
```

File: Documentation/Security/GuidelinesIntegrators/InstallTool.rst [hunk 3]
```diff
@@ -150,5 +150,22 @@ information, cache content, etc. and you should clear all caches after
...
+
+
+..  code-block:: php
+
+    use  \TYPO3\CMS\Core\Crypto\Random;
```
Comment (brotkrueml) [inline Documentation/Security/GuidelinesIntegrators/InstallTool.rst:169]:
```suggestion
    use \TYPO3\CMS\Core\Crypto\Random;
```

Comment (brotkrueml) [reply Documentation/Security/GuidelinesIntegrators/InstallTool.rst:169]: I agree with you (see Slack) we should provide a command for that, a button in the backend, or at least a small PHP snippet.

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/Security/GuidelinesIntegrators/InstallTool.rst (hunk 3 above)
Comment (linawolf) [reply Documentation/Security/GuidelinesIntegrators/InstallTool.rst:169]: But for now this is better then nothiing?

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**